### PR TITLE
Refactor _admin partial to _extra_links partial.

### PR DESCRIPTION
**Pull request by ashercodeorg**

### CODE REVIEW

File: dashboard/app/views/shared/_extra_links.html.haml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    extraLinksBox.toggle();
+    if (extraLinksBox.is(":visible")) {
+      // Clear the cookie using a blank value and a date in the past.
+      document.cookie = "hide-extra-links-menu=; expires=Thu, 01 Jan 1970 00:00:00 UTC;";
+    } else {
+      // Set the cookie and make it expire far into the future.
+      document.cookie="hide-extra-links-menu=true; expires=Fri, 31 Dec 9999 23:59:59 GMT;";
+    }
+    return false;
+  }
```
Comment (joshlory) [inline dashboard/app/views/shared/_extra_links.html.haml:31]: Now might be a good time to pull this JavaScript out of the Haml view.

Comment (ashercodeorg) [inline dashboard/app/views/shared/_extra_links.html.haml:31]: SGTM. Is `code-studio/src/js/extraLinks.js` the appropriate destination file?

**Comment by ashercodeorg** [comment]
Note that I'm reviving this PR. Guidance on moving the JS welcome.

File: dashboard/app/views/shared/_extra_links.html.haml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+.extraLinks
```
Comment (joshlory) [inline dashboard/app/views/shared/_extra_links.html.haml:1]: Need to also update the SCSS file? https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/assets/stylesheets/admin.css.scss

File: dashboard/app/views/shared/_extra_links.html.haml (hunk 1 above)
Comment (joshlory) [inline dashboard/app/views/shared/_extra_links.html.haml:31]: I think so.  Ideally it would be a separate bundle (see e.g. the PLC bundle) but it's a small enough file it's not a huge deal.

Comment (ashercodeorg) [inline dashboard/app/views/shared/_extra_links.html.haml:31]: Having issues verifying locally that my moving the JS outside of the HAML doesn't break anything (likely, given my knowledge of JS (in our system)). Is it OK if I punt moving the JS to another PR?

Comment (joshlory) [inline dashboard/app/views/shared/_extra_links.html.haml:31]: Yep I think that's fine.

**Comment by joshlory** [comment]
LGTM
